backend/app/views/backoffice/projects/_filters.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "backoffice/shared/filters", q: q do |f| %>
+  <%= render partial: "backoffice/shared/full_text_filter", locals: { f: f } %>
+  <div class="w-30 inline-block mx-3">
+    <%= f.input :status_eq, as: :select, required: false, label: false, include_blank: I18n.t("backoffice.common.status"),
+                collection: Project.statuses.map { |k, v| [I18n.t("activerecord.attributes.project.statuses.#{k}"), v] } %>
```

Comment:
I would add enum model for `ProjectStatus`, and keep the translations there. Maybe frontend will need this as well.